source/refined-github.css

```diff
@@ -1,7 +1,8 @@
 :root {
-	--github-green: var(--color-fg-success, #1a7f37);
-	--github-red: var(--color-fg-danger, #cf222e);
-	--github-border-color: var(--color-border-muted, #d8dee4);
+	--rgh-green: var(--color-fg-success, #1a7f37);
+	--rgh-red: var(--color-fg-danger, #cf222e);
+	--rgh-border-color: var(--color-border-muted, #d8dee4);
+	--rgh-background: var(--color-canvas-default, var(--color-bg-default, var(--color-bg-canvas, #fff)));
```

Comment:
Actually we could just drop `--color-bg-default` because this var probably never existed.

It seems some variables match the new class names, but some don't, e.g. `.color-fg-success` but `--color-success-fg` :man_shrugging: